### DISCUSSION TITLE
Big ass update

### DISCUSSION THREAD

**Discussion by HiddenStriker** [Pryaxis/TShock#2497] [General]
Well, there's a brand new big ass update. The new version of Terraria doesn't work with the current version of TShock anymore :-(. I know it's brand new, but I'm curious, what do you guys wager the ETA will be for a TShock update?

**Comment by hakusaro** [comment]
> Big ass update

**Reply by HiddenStriker** [reply]
Yeah it has a boss, items, pets, and all that noise. I wanted to emphasize that it will probably take a lot of effort to update for it, since people have a habit to misunderstand I didn't want to be understood as demanding and get a boilerplate "we do this for free" answer and wanted to word it nicely. Not sure if it will take you guys a long time or not at all.

**Reply by DavidSenpai** [reply]
There's a new crossover update
https://store.steampowered.com/news/app/105600/view/3093403547496230162

terraria: we’re never updating the game again.
company: we will give u lots of money to colab with us.
terraria: introducing our latest update;

**Comment by hakusaro** [comment]
Hi! Please see: https://github.com/Pryaxis/TShock/discussions/2499

Thanks for your continued help in testing!

**Comment by Minenash** [comment]
Just so I know how temporary of a solution I need, would an update of this scale be closer to weeks or months?

**Reply by hakusaro** [reply]
Did you see this link? https://github.com/Pryaxis/TShock/discussions/2497#discussioncomment-1673912

**Reply by Minenash** [reply]
I did, but I didn't know how much it was updated. Like, for example, updated to run, but not necessarily to use. Or working but not stable (I've had a few things where the getting it to work was quick, but getting it to a point where I could recommend people to use it on stuff they care about took a lot longer)

Also sorry, I know almost nothing about terraria, just hosting the server on a pi for them, so I don't have a good world to test on my own (if I even knew what to test)

**Reply by hakusaro** [reply]
@Minenash for updates like this and on our current stack it’s mostly usable honestly.

**Reply by Minenash** [reply]
Cool! Thanks for the quick update and response!

**Comment by hakusaro** [comment]
It’s out! https://github.com/Pryaxis/TShock/releases/tag/v4.5.6

**Reply by DavidSenpai** [reply]
Thank you 
ill keep an eye out for bugs etc.
